Mergin/utils.py

```diff
@@ -408,13 +408,12 @@ def get_new_qgis_project_filepath(project_name=None):
     return None
 
 
-def unsaved_project_check(force_saving=False):
+def unsaved_project_check():
     """
     Check if current QGIS project has some unsaved changes.
     Let the user decide if the changes are to be saved before continuing.
-    :return: True if previous method should continue, False otherwise.
-    If 'force_saving' flag is set to True then changes have to be saved before continuing.
-    :type: boolean
+    :return: True if previous method should continue, False otherwise and replay button (or None if no changes detected)
+    :type: boolean, QMessageBox.Yes|QMessageBox.No|QMessageBox.Cancel|None
```

Comment:
Maybe we should rename Cancel to Ignore for better clarity? If I read code correctly Cancel does not cancel anything, it just tells to leave changes and pretend that project and layers are in the clean state.

Reply:
Actually "Cancel" button have a reject role, so it should result with not displaying Diff dialog.

Reply:
Done.